tests/schema/test_docstring.py

```diff
@@ -0,0 +1,293 @@
+import textwrap
```

Comment:
Great tests! I have a couple of requests tho:
1. Can the tests be made a bit more unit-y, where we test the docstring type of one type at a time? I.e. a test for enums, then a test for resolvers, then a test for objects, etc. 
2. Explicit tests that ensure docstring is/is not exposed for each of the `DescriptionSource` choices.
3. What do you think about adding some for a few of the other formats `docstring_parser` supports? Just so that viewers looking at the tests for examples understand that multiple formats are supported